### DISCUSSION TITLE
I'm a novice, can anyone help get FanControl to work with new AMD GPU?

### DISCUSSION THREAD

**Discussion by Heffstar** [Rem0o/FanControl.ADLX#41] [Q&A]
Hi there, I've never used github before and am very keen to get FanControl working with my new AMD GPU. 
Is there a guide or instructions on how to get this plug in to work? I read the 'readme' and was lost at download the binaries and compile the solution... Any help would be much appreciated. Thanks.

**Comment by nikolarun** [comment]
Get it from https://ci.appveyor.com/project/Rem0o/fancontrol-adlx/build/artifacts 
and copy into the plugin directory

**Comment by nikolarun** [comment]
here is the link - https://github.com/Rem0o/FanControl.ADLX

**Comment by Heffstar** [comment]
Thank you. I put the files in the plug-in folder. And then launched FanControl and added the plug-in through the settings. The GPU info displayed for a second, but now FanControl is unresponsive and crashes. I will just revert to not using FanControl for GPU. Thanks very much for the response and assistance.

**Comment by nikolarun** [comment]
do you have the latest version of fan control ?

**Comment by Heffstar** [comment]
Yeah, I did. I couldn't get FanControl to work again after that. Ended up deleting the files, uninstalling and reinstalling to get it to work again. Thanks for your help, was worth a shot, but I won't bother now. If I end up with a Nvidia card again will get it sorted.
